### ISSUE TITLE
The latest tag name is wrong

### ISSUE THREAD

**Issue by zingimmick** [rectorphp/rector#3655]
# Question
The latest tag named as `vO.7.43`, should be `v0.7.43`.

**Comment by TomasVotruba** [comment]
I don't get it. It's 2 same versions. Could you rephrase?

**Comment by zingimmick** [comment]
The first is letter O, and second is zero

**Comment by TomasVotruba** [comment]
Oh, not sure how that happened. 

I'll fix it tonight

**Comment by TomasVotruba** [comment]
Fixed: https://github.com/rectorphp/rector/releases/tag/v0.7.43

Thanks for reporting
